src/screens/FeedScreen/elements/BlurredLayer.js

```diff
@@ -109,8 +109,8 @@ const styles = StyleSheet.create({
     fontWeight: 'normal',
     fontSize: normalizeFontSizeByWidth(12),
     lineHeight: 18,
-    color: COLORS.white,
-    borderColor: COLORS.white,
+    color: COLORS.almostBlack,
+    borderColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in color for both the text and border from `COLORS.white` to `COLORS.almostBlack` could potentially impact the visibility and contrast of the element against its background. Please verify that these changes align with the intended design and maintain sufficient contrast for accessibility.
```diff
-    color: COLORS.white,
-    borderColor: COLORS.white,
+    color: COLORS.almostBlack,
+    borderColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->